### DISCUSSION TITLE
2019/06/cara-enkripsi-file-atau-aplikasi-dengan

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#36] [General]
# 2019/06/cara-enkripsi-file-atau-aplikasi-dengan

Enkripsi file dengan menggunakan crpyto-js sebagai pengaman file dengan sandi sebagai pencegahan pencurian berkas.

https://www.mantankode.net/2019/06/cara-enkripsi-file-atau-aplikasi-dengan.html?m=1

**Comment by yusufhikam** [comment]
pakai algoritma apa kak?

**Reply by madi10** [reply]
program ini bukan buatan saya, awal mulanya lagi cari cara enkripsi url dengan crypto-js. dan ketemu cara enkripsi file dengan JavaScript (crypto-js), karena menurut saya ini penting dan berguna. jadi saya ulas dengan versi bahasa dan pemahaman saya.

jika tertarik dengan algoritma nya bisa ke web ini -> https://tutorialzine.com/2013/11/javascript-file-encrypter

maaf baru balas yah @yusufhikam
